pkg/nimble/netif/include/nimble_netif.h

```diff
@@ -103,6 +104,46 @@ extern "C" {
 #define NIMBLE_NETIF_MTU            (1280U)
 #endif
 
+/**
+ * @brief   Flags for enabling legacy advertisement and high-duty cycle mode
+ *          when accepting incoming connections
+ */
+enum {
+    NIMBLE_NETIF_FLAG_LEGACY    = 0x01,     /**< use legacy advertising mode */
+    NIMBLE_NETIF_FLAG_HD_MODE   = 0x02,     /**< use high duty cycle mode, only
+                                             *   valid for direct advertising */
+};
+
+/**
+ * @brief   Parameter set used to configure accepting connections (advertising)
+ */
+typedef struct {
+    uint8_t flags;              /**< flags */
+    uint8_t channel_map;        /**< specify custom channel map */
+    uint8_t own_addr_type;      /**< specify our own address type to use */
+    int8_t tx_power;            /**< specify TX power to be used */
+    uint32_t adv_itvl_ms;       /**< advertising interval [ms] */
+    uint32_t timeout_ms;        /**< stop accepting after this time [ms] */
+    nimble_phy_t primary_phy;   /**< primary PHY mode */
+    nimble_phy_t secondary_phy; /**< secondary PHY mode */
```

Comment:
Can you align them as suggested by @benpicco? or maybe I missed your answer to that request?
```suggestion
    uint32_t adv_itvl_ms;       /**< advertising interval [ms] */
    uint32_t timeout_ms;        /**< stop accepting after this time [ms] */
    uint8_t flags;              /**< flags */
    uint8_t channel_map;        /**< specify custom channel map */
    uint8_t own_addr_type;      /**< specify our own address type to use */
    int8_t tx_power;            /**< specify TX power to be used */
    nimble_phy_t primary_phy;   /**< primary PHY mode */
    nimble_phy_t secondary_phy; /**< secondary PHY mode */
```

Reply:
I did address this. This current version of the code has the same size than the suggested order...